lib/lambda_ethereum_consensus/store/blocks.ex

```diff
@@ -0,0 +1,68 @@
+defmodule LambdaEthereumConsensus.Store.Blocks do
+  @moduledoc false
+  alias LambdaEthereumConsensus.Store.BlockStore
+
+  use GenServer
+
+  @ets_block_by_hash __MODULE__
+
+  ##########################
+  ### Public API
+  ##########################
+
+  def start_link(_opts) do
+    GenServer.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  def store_block(block_root, signed_block) do
+    cache_block(block_root, signed_block)
```

Comment:
Let's put this inside the cast for now and make the ets protected. We can think about optimizing writes later on. I think (although we can check) that we should get way more reads than writes.

Reply:
Discussed offline.